src/main/resources/application.yml

```diff
@@ -5,6 +5,13 @@ api:
   ero-management:
     url: ${API_ERO_MANAGEMENT_URL}
 
+sftp:
+  host: ${SFTP_HOST}
+  user: ${SFTP_USER}
+  password: ${SFTP_PASSWORD}
+  private-key: ${SFTP_PRIVATE_KEY}
+  print-request-upload-directory: ${SFTP_PRINT_REQUEST_UPLOAD_DIRECTORY}
+
```

Comment:
Question: In my next PR I will be adding another property (something like this)

`print-response-download-directory: ${SFTP_PRINT_RESPONSE_DOWNLOAD_DIRECTORY}` which will point to say  `EROP/Dev/OutBound` , 

Can we reuse the same `sftpRemoteFileTemplate` within SftpConfiguration` configuration class or would I need another bean for outbound ?

Reply:
Update: I will create another `sftpOutboundTemplate` bean

Reply:
Agree - I think the outbound and inbound are separate channels 👍 (and beans)